### DISCUSSION TITLE
Problem beim Speichern

### DISCUSSION THREAD

**Discussion by mikerway** [tbnobody/OpenDTU#1737] [General]
Hallo,
wenn ich unter Einstellungen - DTU das Abfrageintervall ändern möchte, kommt die Fehlermeldung "Die Frequenz muss zwischen 860250000 und 923500000 kHz liegen und ein vielfaches von 250kHz betragen!" und es wird nichts gespeichert, alles bleibt so wie es ist.
WR HM-1500 schon neu gestartet, ebenso opendtu, ist ne gekaufte von AllianceApps, alles andere scheint normal zu funktionieren...
Was tun?

**Comment by tbnobody** [comment]
z.B. die Issues lesen oder das tun was nach dem Firmware Update angezeigt wird (Webseite neu laden)

Siehe auch #1651 oder #1662

**Comment by mikerway** [comment]
Wenn ich das richtig verstanden habe, dann sollte beim NRF24 die Sendeleistung runter gesetzt, ein anderes Netzteil verwendet oder ein Kondensator eingebaut werden?
Kondensator ist schon verbaut, 3 verschiedene Netzteile getestet, ebenso Sendeleistung versucht auf minimum zu stellen und Cache geleert, Fehlermeldung bleibt...
Ist die opendtu dann ein Fall für den Mülleimer?

**Comment by tbnobody** [comment]
Welche OpenDTU Version verwendest du?  Und teile doch mal bitte einen Screenshot von dem "Einstellungen" --> "DTU" Dialog.

**Comment by mikerway** [comment]
v24.1.26, habe aber schon andere probiert
![Screenshot_20240212-194816](https://github.com/tbnobody/OpenDTU/assets/159808123/f53c15e1-28ce-4423-81a0-8c14753fae82)

**Comment by mikerway** [comment]
Problem vorerst gelöst, habe auf Werkseinstellungen zurück gestellt, alles neu eingegeben und nun lassen sich die Einstellungen wieder speichern.
